core/src/main/java/com/klaytn/caver/wallet/KeyringContainer.java

```diff
@@ -40,11 +41,19 @@
      */
     Map<String, AbstractKeyring> addressKeyringMap = new HashMap<>();
 
+    /**
+     * The KeyringFactoryWrapper instance
+     * This is added to improve "SDK User Experience"
+     * which means giving developers similar development experience with caver-js
+     */
+    public KeyringFactoryWrapper keyring;
 
     /**
      * Creates KeyringContainer instance
      */
-    public KeyringContainer() {}
+    public KeyringContainer() {
+        keyring = new KeyringFactoryWrapper();
```

Comment:
This should be applied in other constructors as well.

Reply:
I missed it 😿 
I'll add initialization to another constructor also

Reply:
I applied your feedback here ^_< [Add missed member variable initialization](https://github.com/klaytn/caver-java/pull/215/commits/75a2b2114b3fafd3951a0ed1ed73732462b3908d)